exercise-src/react-native/07-debugging/01-solution/package.json

```diff
@@ -11,7 +11,8 @@
     "prettier": "prettier --check *.ts *.tsx *.js",
     "depcheck": "depcheck .",
     "precheck": "npm run typecheck && npm run eslint && npm run prettier && npm run test && npm run depcheck",
-    "devtools": "react-devtools"
+    "devtools": "react-devtools",
+    "debugger": "npx react-native start --experimental-debugger"
```

Comment:
Two things:

1) In general I would keep all the scripts in alphabetical order. I know they’re not, so at a minimum I would add `debugger` somewhere that’s not the end.

2) Does this change need to be made to the later `package.json` files too, or is this the last one?

Reply:
Yes it changes for later package.json